app/src/main/kotlin/com/wire/android/ui/home/HomeTopBar.kt

```diff
@@ -42,9 +42,11 @@ fun HomeTopBar(
     val scrollDownState = viewModel.scrollDownFlow.collectAsState(false)
     val firstLineElevation = if (!isSearchable || scrollDownState.value) dimensions().topBarElevationHeight else 0.dp
 
-    Box(
-        Modifier.background(Color.Transparent)
-    ) {
+    LaunchedEffect(viewModel) {
```

Comment:
hmmm... what did you try to achieve with it using viewModel as a key ?

Reply:
Added some inline comment to clarify it